## The big question of cell size

### Wallace F Marshall

For well over 100 years, cell biologists have been wondering what determines the size of cells. In modern times, we know all of the molecules that control the cell cycle and cell division, but we still do not understand how cell size is determined. To check whether modern cell biology has made any inroads on this age-old question, *BMC Biology* asked several heavyweights in the field to tell us how they think cell size is controlled, drawing on a range of different cell types. The essays in this collection address two related questions - why does cell size matter, and how do cells control it.

Why do cells care how big or small they are? One reason cell size matters is that the basic processes of cell physiology, such as flux across membranes, are by their nature dependent on cell size. As a result, changes in cell volume or surface area will have profound effects on metabolic flux, biosynthetic capacity, and nutrient exchange. A second reason is that the basic machinery of cell division in eukaryotes relies on microtubules, both to form the mitotic spindle and position it properly relative to the cortex. Because of the dynamic properties of microtubules, they are able to probe a limited range of lengths, and if cells get too big or too small, the mitotic apparatus may have difficulty working. Very small cells could not form a proper spindle, and very large cells could not coordinate their divisions during cleavage. This idea is elaborated in essays by Frankel and by Kimura, who discuss the apparent upper and lower limits on cell size with respect to cell division machinery. Finally, in both animals and plants, cells must fit together like puzzle pieces to form tissues and organs, and that means that a cell has to have a size appropriate to its position within the overall tissue, a topic discussed by Wallingford in the context of animal development.

Given that cell size is important, how can a cell control how big it is? In terms of 'design principles' for a size control system, the most fundamental question is whether cells need to know how big they are in order to regulate size. The simplest model is one in which cell mass grows at some rate determined by biosynthetic reactions (the rate could be dependent on cell size or not), and as they are growing, the cells divide at some constant frequency set by the cell cycle clock. Such a scheme would not require cells to ever actually know how big they are, but as discussed by Swaffer, Wood, and Nurse for yeast cells, experimental evidence rejects this simple model and suggests instead that cells can measure their own size and regulate the timing of cell division accordingly. This leads to the idea that cells can measure size, possibly by reading out intracellular gradients. But as discussed by Young and by Qu and Roeder, mechanical properties of the cell surface and of cytoskeletal elements can also play a role in determining size.

At the end of the day regulation of cell size may prove to be the combined result of several mechanisms operating in parallel, and that may be one reason it has been hard to study.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## Bacteria: appearances matter!

### Kevin D Young

The most obvious characteristic of bacteria is that they are small. Really small. As in requiring microscopes of high magnifying and resolving power to see them. So it surprises people to learn that the volume of these normally tiny cells can differ by as much as 106- to 108-fold, from the tiniest 0.2 μm cells of the *Pelagibacter* SAR11 clade that fills the oceans [[1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B1)] to the monstrous genera *Thiomargarita* and *Epulopiscium* in which some species measure over 600 to 700 μm in length or diameter and are visible to the naked eye [[2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B2)-[4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B4)]. Of course, large bacteria are an extreme minority, with most known bacteria falling somewhere between 0.4 and 2 μm in diameter and 0.5 and 5 μm in length (though many grow as filaments that can be tens or hundreds of times this long). Another conceit is that bacteria are boring, at least in morphological terms. But this is just because most of us rarely encounter bacteria outside of what are usually brief episodes of disease, and the shapes of these common bacteria are admittedly pretty lame, being, as they are, no more than tiny cylinders. However, on a more global scale, bacterial shapes range from the plain (rods, spheres, strings) to the outlandish (branched, curved, coiled, spiraled, star-shaped), to the truly bizarre (fluted and tentacled) [[5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B5)]. Given this range of possibilities, what determines the morphology of any one bacterium?

The first determinant is, as always, evolutionary. Bacteria cope with at least six fundamental selective forces that have some degree of control over the size that will best suit them to survive in particular environments. Specifically, bacteria adopt certain sizes and shapes so they can import nutrients most efficiently, meet requirements imposed by cell division, attach themselves to external surfaces, take advantage of passive dispersal mechanisms, move purposefully to pursue nutrients or avoid inhibitors, or avoid predation by other organisms [[5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B5),[6](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B6)]. Fundamental to all these considerations is that bacteria must accumulate nutrients that reach them by diffusion alone [[7](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B7)]. A basic tenet is that for such cells to exist the ratio of their surface area to cytoplasmic volume has to be quite high. Therefore, to maximize this ratio, most bacteria produce cells in the 0.2 to 10 μm size range and some organisms extrude long, exceedingly thin appendages to harvest nutrients present in low concentrations [[8](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B8)]. Because of this reliance on diffusion, those bacteria that reach near-millimeter size do so by employing clever morphological tricks. For example, some deploy their cytoplasm as a thin film around the outer rim of a large internal vacuole, creating a cell that looks very much like the skin of a balloon [[2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B2),[9](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B9)]. Others localize tens of thousands of chromosomes around the periphery of their cytoplasm, in near contact with the cell surface, so that each genomic equivalent 'governs' a volume approximately equal to that of a more normal, smaller cell [[4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B4)]. Where a particular bacterium will eventually land in this size universe depends on other selective forces, which basically revolve around a bacterium's need to put itself in position to reach any nutrients at all versus the need to defend itself against becoming a nutrient for others.

The second determinant of bacterial morphology is mechanical, a factor that encompasses the biochemical mechanisms that do the heavy lifting of constructing cells of defined sizes and shapes. The current consensus is that morphology is determined primarily by molecular machines that synthesize the rigid cell wall. Three major types of machines are available. One, directed by the protein FtsZ, is responsible for nucleating the process of cell division and is shared by all bacteria, while the other, directed by the protein MreB and its homologues, is responsible for cell elongation in rod-shaped bacteria [[10](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B10)-[13](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B13)]. The third, first recognized by the activity of the CreS (crescentin) protein of *Caulobacter crescentus*, is responsible for creating the curved cells of this organism and the more regular shapes of other bacteria [[14](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B14),[15](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B15)]. In a series of conceptual surprises, it was realized that FtsZ is a homologue, and perhaps progenitor, of the eukaryotic cytoskeletal protein tubulin [[16](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B16),[17](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B17)], that MreB is a homologue of actin [[18](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B18),[19](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B19)], and that CreS and its relatives are homologues of intermediate filaments, a third class of eukaryotic cytoskeleton proteins [[14](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B14),[15](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B15)]. Though the structural similarities are clear, these proteins have been co-opted to perform different functions in bacteria. One last curiosity deserves mention: some classic metabolic enzymes also moonlight as cytoskeletal filaments that affect bacterial shape, a discovery with potentially far-reaching implications [[20](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B20),[21](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B21)]. Finally, these basic tools can be modified, supplemented or differentially regulated to create morphologies from the simple to the quite complex.

There is room here to give only three brief examples of how rod-shaped bacteria control their overall size by varying cell length. The first involves *Escherichia coli*, a plain cylindrical rod that is normally about 1 μm in diameter and 2 μm long. In this organism, the future division site is determined by at least two mechanisms, each of which inhibits the polymerization or function of FtsZ and thus regulates when and where cell division occurs. First, driven by the MinD and MinE proteins, the MinC inhibitor oscillates back and forth between the two polar ends of the cell, taking approximately 1 to 2 minutes per cycle [[22](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B22),[23](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B23)]. This behavior creates a time-averaged MinC concentration gradient that is highest at the poles and lowest near mid-cell. As the cell elongates, the concentration near the cell's center is reduced until it becomes so low that FtsZ can polymerize and initiate cell division. Therefore, cell size (as measured by length) is determined by the amount of MinC - larger amounts produce longer cells. Conceptually, this is eerily similar to the mechanism that regulates cell length in rod shaped fission yeast, as described by Swaffer *et al*. in this Forum article (below). Though there are biochemical differences, in this eukaryote cell length is regulated by a concentration gradient of Pom1 that is highest at the poles of a growing cell. Division is therefore inhibited until the cells become long enough so that the concentration of Pom1 at the cell center drops low enough to allow division.

The second way *E. coli* regulates cell length is by a 'nucleoid occlusion' mechanism [[24](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B24)]. Here, the SlmA protein binds to specific DNA sequences, and the SlmA-DNA complex prevents cell division by inhibiting FtsZ. Interestingly, SlmA binding sites are distributed around the chromosome except near the area where DNA replication terminates. During chromosomal segregation the two origins are pulled to either pole, and the two termination regions remain near the cell center, where they are the last to be replicated and separated. This means that as replication ends and when segregation is almost complete there will be a dearth of SlmA near mid-cell, at which time FtsZ will no longer be inhibited and can trigger division. Again, note how similar this is to the kind of mechanism that may explain how chromosomal ploidy determines cell length in yeast (see the contribution from Swafer *et al*. in this Forum article, below).

Recently a third, and surprising, mechanism was discovered by which cell length is tied to the metabolic status of the cell. *Bacillus subtilis*, a rod shaped bacterium about 1 to 2 μm in diameter and 5 to 10 μm in length, is longer when incubated in a nutrient-rich medium and shorter when nutrients are limited. Although it sounds simple, the question of how bacteria accomplish this has persisted for decades without resolution, until quite recently. The answer is that in a rich medium (that is, one containing glucose) *B. subtilis* accumulates a metabolite that induces an enzyme that, in turn, inhibits FtsZ (again!) and delays cell division. Thus, in a rich medium, the cells grow just a bit longer before they can initiate and complete division [[25](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B25),[26](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B26)]. These examples suggest that the division apparatus is a common target for controlling cell length and size in bacteria, just as it may be in eukaryotic organisms.

In contrast to the regulation of length, the MreB-related pathways that control bacterial cell width remain highly enigmatic [[11](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B11)]. It is not just a question of setting a specified diameter in the first place, which is a fundamental and unanswered question, but maintaining that diameter so that the resulting rod-shaped cell is smooth and uniform along its entire length. For some years it was thought that MreB and its relatives polymerized to form a continuous helical filament just beneath the cytoplasmic membrane and that this cytoskeleton-like arrangement established and maintained cell diameter. However, these structures seem to have been figments generated by the low resolution of light microscopy. Instead, individual molecules (or at the most, short MreB oligomers) move along the inner surface of the cytoplasmic membrane, following independent, almost perfectly circular paths that are oriented perpendicular to the long axis of the cell [[27](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B27)-[29](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B29)]. How this behavior generates a specific and constant diameter is the subject of quite a bit of debate and experimentation. Of course, if this 'simple' matter of determining diameter is still up in the air, it comes as no surprise that the mechanisms for creating even more complicated morphologies are even less well understood.

In short, bacteria vary widely in size and shape, do so in response to the demands of the environment and predators, and create disparate morphologies by physical-biochemical mechanisms that promote access to a huge range of shapes. In this latter sense they are far from passive, manipulating their external architecture with a molecular precision that should awe any contemporary nanotechnologist. The techniques by which they accomplish these feats are just beginning to yield to experiment, and the principles underlying these abilities promise to provide valuable insights across a broad swath of fields, including basic biology, biochemistry, pathogenesis, cytoskeletal structure and materials fabrication, to name but a few.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## The puzzling influence of ploidy

### Matthew Swaffer, Elizabeth Wood, Paul Nurse

Cells of a particular type, whether making up a specific tissue or growing as single cells, often maintain a constant size. It is usually thought that this cell size maintenance is brought about by coordinating cell cycle progression with attainment of a critical size, which will result in cells having a limited size dispersion when they divide. Yeasts have been used to investigate the mechanisms by which cells measure their size and integrate this information into the cell cycle control. Here we will outline recent models developed from the yeast work and address a key but rather neglected issue, the correlation of cell size with ploidy.

First, to maintain a constant size, is it really necessary to invoke that passage through a particular cell cycle stage requires attainment of a critical cell size? If cells grow linearly - that is, the rate at which they accumulate mass in unit time is constant regardless of the mass of the cell - and if the time between successive cell divisions is maintained by a fixed timer, then cells will maintain size homeostasis. In successive generations all cells will slowly tend towards an average size [[30](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B30)]. However, work from both fission and budding yeast has shown this not to be the case [[31](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B31)]. Firstly, the variation in sizes at division of both yeast species is too small to be accounted for by such a process [[31](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B31)]. Secondly, cell cycle arrest of fission yeast results in enlarged cells that exhibit significantly shortened subsequent cycles [[32](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B32)]. This rapid reversion to the original cell size indicates the presence of a size correction mechanism. Similarly, in budding yeast, cells born smaller than normal spend longer in G1 until they reach a critical size [[33](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B33)]. There is also no evidence to suggest that yeast cells accumulate mass in a simple linear way for extended periods of time [[34](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B34)-[37](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B37)]. Therefore, there is a mechanism that monitors cell size and uses this information to regulate progression through events of the cell cycle. In the case of fission yeast this occurs primarily during G2 [[38](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B38),[39](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B39)] but can operate in G1 [[39](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B39),[40](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B40)], and for budding yeast it occurs during G1 [[33](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B33),[41](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B41)].

Two different molecular mechanisms for size control have been proposed for the two yeasts. In fission yeast, Cdc2 (Cdk1) kinase activity drives entry into mitosis and thus determines the length of G2 [[42](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B42)]. Wee1 catalyzes inhibitory phosphorylation of Cdc2 on Tyr15 and is antagonized by the phosphatase Cdc25 [[42](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B42)-[46](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B46)]. Cell size information is transduced via Cdr1 and Cdr2, inhibitors of Wee1 that localize to cortical nodes at the center of the cell [[47](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B47)]. Pom1 is a kinase that inhibits Cdr1/Cdr2, thus alleviating inhibition of Wee1 [[47](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B47),[48](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B48)]. A gradient of Pom1 emanating from the cell tips inhibits G2/M until cells reach a critical length, and as cells are rod-shaped this is correlated with cell size. Pom1 is delivered to and associates with the plasma membrane at the cell ends. Pom1 autophosphorylation results in membrane dissociation, generating the Pom1 gradient, with a high concentration at the tips decreasing towards the cell center [[49](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B49)]. In a small early G2 cell there is sufficient Pom1 at the cortical nodes to inhibit Cdr1/Cdr2, preventing mitotic entry (Figure [(Figure1a).1a](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F1/)). As cells grow and elongate, Pom1 concentration at the medial site becomes progressively lower, Wee1 is eventually inhibited by Cdr1/Cdr2, and cells commit to mitotic division [[47](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B47),[48](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B48)] (Figure [(Figure1b).1b](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F1/)). In this way the size of the cell regulates mitotic entry. However, this is unlikely to be the whole story because in cells where Cdc2 Tyr15 phosphorylation is prevented from occurring (thus bypassing Pom1-mediated regulation) cell size homeostasis is maintained, albeit with a broader size distribution [[50](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B50)]. This indicates other unknown mechanisms operate to measure size and integrate this information into cell cycle control.



![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-1.jpg)

(https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-1.jpg)



[Figure 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F1/)

**The Pom1 gradient model for length sensing in fission yeast**. **(a)** An early G2 cell. Pom1 protein emanating from the cell tips (purple) inhibits Cdr1/Cdr2 in the cortical nodes (light blue circles). Wee1 is therefore active and carries out inhibitory phosphorylation of Cdc2. (**b)** A late G2 cell. Pom1 concentration at the medial site is decreased. At a critical threshold, Cdr1/Cdr2 are no longer inhibited (dark blue circles) and so the Wee1 inhibition of Cdc2 is lifted. The active CDK drives mitotic entry.

In budding yeast, size control operates at 'start', a G1 event that commits the cell to cycle at a given size [[33](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B33),[41](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B41)]. This is thought to operate by a protein synthesis-rate sizer mechanism involving the G1 cyclin Cln3 [[31](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B31),[51](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B51),[52](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B52)]. Cln3 is a dose-dependent activator of start [[53](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B53)-[55](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B55)], and is rapidly degraded [[56](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B56)]. Its high turnover rate means that the amount of Cln3 should be a direct reflection of the current rate of protein synthesis within the cell [[57](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B57)]. Since the number of ribosomes is indicative of cell size, protein synthesis rate will correlate with cell size. Therefore, only once a certain cell size is reached will there be sufficient Cln3 to drive the transition through start. This is thought to involve the activation of the transcription factor SBF by Cln3-CDK-mediated inhibition of Whi5, an SBF inhibitor [[58](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B58),[59](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B59)]. A major problem with using protein synthesis rate as a proxy for cell size is that as the cell gets larger and more Cln3 is produced, the corresponding increase in cell volume should dilute out the protein, keeping it at a constant concentration. This appears to be the case for Cln3, as its relative abundance does not significantly change as cells grow during G1 [[55](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B55)]. To overcome this problem it has been proposed that Cln3 import into a nucleus of fixed size would allow the cell to 'measure' the absolute amount of Cln3 [[51](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B51)]. However, it has been shown for both budding and fission yeast that nuclear volume increases with the size of the cell [[60](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B60),[61](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B61)], so this particular model remains incomplete and still requires a fixed 'standard' against which to measure Cln3.

Both of these models are interesting but, at least in their simplest form, they cannot account for a close to universal aspect of cell size: that is, the almost directly proportional increase in size at division that is observed as ploidy increases [[62](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B62),[63](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B63)]. This relationship holds within a ploidy series of a single species, as well as across species [[60](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B60),[64](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B64)]. This observation indicates that somehow cells can monitor their ploidy and integrate this information into the cell size-monitoring mechanisms. In general terms we envisage two types of model by which this might operate: either the cell makes a specific amount of a critical component according to ploidy, or it measures the amount of a given factor against ploidy.

An example of the former model would be if the transcript of the critical component were produced as a single pulse at a specific time in the cell cycle. The size of this burst of transcription would be a direct function of copy number, and as such a reflection of ploidy. If the gene product is stable and acts to inhibit division, as a cell grows, this fixed amount will be diluted down. Below a certain threshold concentration, its inhibitory effect is alleviated and division is permitted [[65](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B65)]. With an increase in ploidy there would be a requirement for a cell to be proportionately larger before the threshold is reached. If any gene were to operate in such a copy number-dependent inhibitory manner, it would be expected that a heterozygous diploid of such a gene would produce half as much protein (with no compensation) and therefore be the size of a haploid. It would be interesting to screen for genes behaving in this manner, but to our knowledge no such gene has yet been described. This may suggest that the model is an oversimplification, and such a mechanism might involve a number of interacting factors resulting in greater redundancy and adding robustness to the system.

An example of the second type of model would be a genomic titration mechanism [[65](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B65),[66](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B66)]. This model invokes a protein that is maintained at a constant concentration, and which binds sites in the genome. As the cell grows larger, the absolute amount of this factor increases and thus more genomic sites become occupied. A critical threshold size is reached when a certain number of sites are bound. The occupancy of these sites could functionally drive a cell cycle transition - for example, by regulating transcription. Alternatively, this threshold could be a point of saturation at which no more sites are available to bind and the factor is free in the nucleoplasm or cytoplasm. The unbound factor could execute a pro-division function, which could even involve binding another DNA sequence for which it has lower affinity. A doubling in ploidy would be accompanied by a doubling in the number of DNA sequences for the protein to bind. This would impart the requirement on the cell to be twice the size before the occupancy threshold is surpassed. These are merely examples of the two types of models that could allow ploidy to regulate cell size and other variants are also possible [[65](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B65)].

Is it possible to modify the two proposed molecular mechanisms described above for yeast size control to take account of the effects of ploidy? With respect to the fission yeast Pom1 gradient model, perhaps the amount of a critical component in the network could be determined by gene copy number. In principle this could be the inhibitor, Pom1, although this is unlikely as deleting one copy of *pom1* in a diploid does not reduce cell size to that of a haploid (Jacqueline Hayles, personal communication). Turning to the Cln3 activator model, it has recently been shown that Cln3 can bind to SBF binding sites across the genome [[67](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B67)]. Introduction of additional SBF binding sites increases cell size at 'start' in a Cln3-dependent manner [[67](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B67)]. It is plausible that this allows the cell to measure the amount of Cln3 against the genome, so only when a fixed number of sites are occupied is division permitted, as per the genome-titration model discussed above. This provides a possible solution to the aforementioned problems with the Cln3 protein synthesis rate sizing mechanism, as well as a means for ploidy to regulate size control directly. Other mechanisms, operating outside of the known size control network, that take account of ploidy could also be envisaged.

The universality of cell size scaling with ploidy means that ploidy should be taken account of when considering cell size-sensing mechanisms. It may also imply that there is conservation of the mechanisms involved, although whether this conservation exists at the level of molecules or of control network architecture remains to be seen.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## Physical limits of cell size for embryonic cell division in *Caenorhabditis elegans*

### Akatsuki Kimura

Early embryos are a good model for studying the relationship between cell size and intracellular organization. Blastomeres can exhibit various sizes during embryonic cell division since cells divide without cell growth during this phase. In addition, embryonic cells are generally large, which makes these cells useful models for microscopic observation. Thus, transparent *Caenorhabditis elegans* embryos represent an ideal model for investigating these relationships [[68](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B68)-[71](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B71)]. Here, I argue, based on prior studies in *C. elegans* and other systems, that cell size may be limited by the physical properties of the cell. In order to proliferate, the cell has to divide, and for faithful cell division, molecular machinery, such as the mitotic spindle, must be constructed at the right position and with the correct size. This may not be accomplished in extremely large or small cells due to the physical properties of macromolecules, such as microtubules and chromosomes.

Positioning of the mitotic spindle at the cell center is critical for symmetric cell division, as it defines the origin of sister chromatid segregation and the position of the cell division plane [[72](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B72)]. The centrosome is a major organizing center of the microtubule cytoskeleton and in animal cells often includes the poles of the mitotic spindle. Centrosomes have the ability to position themselves at the cell center [[73](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B73)], enabling the mitotic spindle also to position at the cell center [[74](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B74),[75](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B75)] (Figure [(Figure2).2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F2/)). This central positioning of the centrosome is accomplished through the function of the microtubule cytoskeleton [[76](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B76),[77](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B77)]. Several mechanisms have been proposed to describe how microtubules bring centrosomes to the cell center, including pushing of the cell cortex and pulling by motor proteins [[74](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B74),[77](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B77)-[88](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B88)]. The mechanisms mediating centrosome centration may differ among species, especially among species with different cell sizes [[83](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B83)]. Recent studies have supported the idea that the cytoplasmic pulling force is a major driving force for centrosome centration in animal cells [[75](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B75),[78](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B78),[81](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B81),[86](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B86),[89](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B89)]. Importantly, for all proposed mechanisms, microtubule-dependent centration of the centrosome must be facilitated by microtubules, which grow from the centrosome and span throughout the cytoplasm to find the geometrical center of the region [[87](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B87),[88](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B88)].



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-2.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-2.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-2.jpg)

[Figure 2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F2/)

**Microtubules in C. elegans**. An image of microtubules in an embryonic cell - astral microtubules from the spindle reach the cell cortex.

If the cell is too large compared to the length of microtubules, the centrosome will not position at the cell center (Figure [(Figure3a,3a](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/), middle panels). Since the microtubules grow and shrink in a stochastic manner known as dynamic instability [[90](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B90)], the mean length of microtubules (*nav*) is defined by the velocities of growth (*v+*) and shrinking (*v-*), the frequency of switching from growth to shrinking (*f+-*) and vice versa (*f-+*), as *nav* ≈ (*v-v+*)/(*v-f+-* - *v+f-+*) [[91](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B91)]. According to this equation, mean lengths are estimated to be in the micron range based on experimentally measured dynamic instability parameters *in vitro* [[92](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B92)] and *in vivo* [[93](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B93)], which is consistent with observed *in vivo* microtubule lengths [[91](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B91),[94](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B94)]. Interestingly, this length is comparable to the size of ordinary animal cells, suggesting that the length scale of a microtubule is related to the size of the cell. Therefore, the length of microtubules may define the upper limit of cell size. This idea has been supported by experimental shortening of microtubule length. When cells were treated with microtubule depolymerizing drugs or partial knockdown of microtubule polymerizing molecules, centrosomes did not reach the cell center and the cell division plane was positioned in an asymmetric manner [[76](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B76),[95](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B95),[96](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B96)]. Interestingly, studies have demonstrated that centrosomes can find the cell center in extremely large cells, such as newly fertilized frog embryos [[97](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B97)] (see the contribution from Dr Frankel in this Forum article, below). In large embryos, a large microtubule aster that expands throughout the cell is formed and centers the centrosome toward the cell center, possibly due to cytoplasmic pulling forces [[89](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B89)]. In an *in vitro* centering experiment using microfabricated chambers, it was demonstrated that efficient elongation of microtubules to reach the boundaries of the chamber was critical for robust positioning of the microtubule aster at the center of the chamber [[88](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B88)]. These studies collectively support the idea that cell radius cannot exceed the maximum length of microtubules for cell proliferation.



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-3.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-3.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-3.jpg)

[Open in a separate window](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/?report=objectonly)

[Figure 3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/)

**Possible scenarios in which centrosome centering (a) and spindle elongation (b) set the upper and lower limit of cell size**. **(a)** If the cell exceeds the upper limit of size, the centrosome, and consequently the mitotic spindle, cannot position at the cell center, leading to nonsymmetrical cell division (middle panels versus left panels). If the cell falls below the lower limit, the centrosome may not stably position at the cell center due to the excess elastic forces of the microtubules (right panels versus left panels). **(b)** If the cell exceeds the upper limit, astral microtubules do not reach the cell cortex, potentially leading to insufficient spindle elongation. If the cell falls below the lower limit, there may not be sufficient space for accurate chromosome segregation compared to the size of the cell's chromosomes.

How are the lower limits of cell size defined? The centrosome may not correctly center if the cell is too small compared to the length of the microtubules. Because of the elastic properties of microtubules, short microtubules generate strong pushing forces when the growing tips encounter obstacles such as the cell cortex. Due to the reactions of these pushing forces, the centrosome will be subjected to strong forces from multiple microtubules, which may destabilize the positioning of the centrosome (Figure [(Figure3a,3a](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/), right panels). This idea is based on an elegant *in vitro* centering experiment using microfabricated chambers combined with theoretical analyses [[79](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B79),[98](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B98),[99](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B99)]. This experiment was performed in a cell-size chamber (a square with about 20 μm on a side), but the length of the microtubules were long due to the lack of shrinking; many microtubules reached the chamber boundaries, exerting pushing forces against the chamber walls. Under these conditions, microtubule asters moved away from the center of the chamber [[79](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B79)] or failed to reposition to the cell center after relocation [[99](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B99)]. This off-center movement was restored by promoting microtubule shrinkage [[99](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B99)]. Shortening of the average length of the microtubule by promoting shrinkage resulted in fewer microtubules reaching the cortex, thereby generating less pushing force, and stably positioned the aster at the geometrical center [[99](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B99)]. This experiment indicated that, in small cells where many short (and thus rigid) microtubules reach the cell cortex, the centrosome cannot stably position at the cell center.

In addition to the physical properties of microtubules such as the length and stiffness, other properties may also define the limits of cell size. Elongation of the mitotic spindle during anaphase is known to depend on cell size; the larger the cell, the longer and faster the spindle elongates. To date, this trend has only been demonstrated in the *C. elegans* embryo [[68](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B68),[69](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B69)]; however, it may represent a general trend in other cells. If a spindle elongates only for a short distance in small cells, the separation of sister chromatids may not be enough to segregate them completely. In the yeast *Saccharomyces cerevisiae*, artificially elongated chromosomes become more condensed at anaphase to ensure complete segregation of the chromatids [[100](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B100)]. This observation indirectly implies that the spindle must elongate to a certain distance in order to segregate chromosomes with a certain size [[101](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B101)]. Within a given genome size, there should be a lower limit to the size of condensed chromosomes, which may further define the lower limit of the elongation of the mitotic spindle, thus defining the size of the cell (Figure [(Figure3b,3b](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/), right panels).

Elongation of the mitotic spindle may define the upper limit of cell size as well. Since elongation is partly driven by pulling astral microtubules from the cell cortex [[102](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B102),[103](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B103)], if astral microtubules do not reach the cell cortex in large cells, the elongation of the spindle is impaired, potentially causing insufficient chromosome segregation (Figure [(Figure3b,3b](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/), middle panels). To my knowledge, there has been no experimental evidence to support this idea so far. However, when the cortical pulling force was impaired in *C. elegans* embryos, we were able to shorten spindle elongation, albeit not completely [[69](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B69)]. Chromosomes appear to manage segregation under these conditions. The identification of genes responsible for the remaining elongation may allow us to test whether impairing the interaction between astral microtubules and the cell cortex leads to chromosome segregation defects.

In this text, I discussed a basic and simplified view of the relationship between cell size and the material properties of macromolecules comprising the mitotic machinery. As the biology always has to deal with diversity, an individual cell may have its own unique mechanism to set cell size and to accomplish cell division. Nevertheless, the diversity in cell size among species is far smaller than that in body size, suggesting common constraints for the majority of cell types. I believe understanding such common constraints will reveal the basic design principle of cell architecture.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## The largest dividing cells: are they alike?

### Joseph Frankel

*'Physical extremes, in this case a very large cytoplasm, are always interesting in biology'* [[97](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B97)]

#### How can we frame a useful inquiry about the upper limits of cell size?

If one asks which cell is the largest, one comes up with a list of highly diverse candidates. Those on the centimeter scale include the ostrich egg measured at 8 cm by EB Wilson [[104](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B104)], the 3 to 5 cm unicellular stalked marine alga *Acetabularia* [[105](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B105)], and various giant shelled (testate) amoeboid denizens of the deep-sea bottom, including the 3 cm 'living fossil' *Gromia sphaerica* [[106](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B106)]. The multinucleate green alga *Caulerpa* is, however, the champion unicellular organism, with a tubular stolon extending to a length of one meter or more, as well as a remarkable degree of internal differentiation and a propensity for rapid vegetative growth [[107](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B107)].

The modes of propagation of *Acetabularia* and *Gromia* are typical of most marine giant single-celled organisms. They do not appear to undergo binary fission, but instead become multinucleate for at least part of their life cycle, after which they produce large numbers of flagellated gametes [[108](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B108),[109](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B109)], each of which includes only a small portion of the overall cytoplasmic mass of the large parent cell [[105](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B105),[108](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B108)]. The same is true for *Caulerpa* during its episodes of sexual reproduction [[110](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B110)]. Thus, these giant cells can endow their progeny with DNA and elementary organelles such as mitochondria and (in algae such as *Acetabularia* and *Caulerpa*) also chloroplasts [[109](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B109),[110](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B110)], but they almost certainly do not transmit any significant portion of their cytoplasmic organization to their individual offspring.

All of these enormous cells have some device for escaping the consequences of their large size when they reproduce, either by cleaving only a small portion of their mass, as in reptilian or bird eggs, or by subdividing all or part of their large cell bodies to produce swarms of diminutive progeny, as in centimeter- and meter-scale marine unicellular organisms such as *Acetabularia* or *Caulerpa*. When producing reproductive cells, such organisms probably do not need to make any global assessment of their overall dimensions and organization. In my view, the interesting upper size limit is the largest size at which a cell can carry out such a global assessment and then make use of this assessment to perpetuate itself by dividing into two daughter cells similar in form and structure to itself. We can then ask what this size limit actually is.

An interesting 'test case' is provided by two amoebae, *Amoeba proteus* and *Chaos chaos*, that are known to be very closely related [[111](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B111)]. *Amoeba proteus* is roughly 500 μM in length when actively moving [[112](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B112)], is uninucleate, and goes through a fairly typical process of mitosis and cytokinesis [[113](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B113)]. By contrast, the giant amoeba, *Chaos chaos* (A.K.A. *Chaos carolinensis, Pelomyxa carolinensis)* measures 1 to 5 mm when extended [[112](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B112)] and is multinucleate. While these nuclei undergo synchronous mitoses [[114](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B114)], the subsequent cell division is very atypical: it simultaneously produces between two and six daughters, and the several hundred nuclei of the parent cell are segregated apparently at random among these division products [[115](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B115)]. This process, called 'plasmotomy' by Kudo [[114](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B114)], is a far cry from the typical mitotic cell division found in the closely related but much smaller uninucleate *Amoeba proteus*.

This example introduces a problem: large cells typically need more DNA than normal-sized cells to support their metabolic and synthetic needs. In extreme cases, they may become multinucleate, in which case the multinucleate condition itself may generate an obstacle to normal cell division. Even more profoundly, the extensive endoreduplication of DNA that is often found in large differentiating cells in plants is closely associated with cessation of cell division [[116](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B116)] (also see the contribution from Qu and Roeder in this Forum article).

Two distinct types of large cells have found different ways of circumventing these obstacles to binary fission. The ciliates do so by possessing a single large polygenomic macronucleus that maintains the vegetative functions of the cell and divides amitotically together with the cell in which it resides. Cleaving eggs do not need multiple nuclei or polytene chromosomes because they possess abundant stored maternal mRNA and (generally) delay the onset of zygotic nuclear transcription, thereby allowing several initial rounds of nuclear division and cytoplasmic cleavage to occur in rapid succession.

I shall here consider these two examples of the consequences of progressive enlargement, in ciliates and in amphibian eggs, the first analyzed using a microsurgical approach and the second following a comparative approach. Comparison of these two cell types indicates that there is a fairly constant upper size limit to whatever organization permits normal cell division, but that the specific organization appears to be markedly different in these two types of cells.

#### A microsurgical approach: grafting *Stentor*

The paradigm for the microsurgical approach has been provided by the largest ciliate, *Stentor coeruleus*, measured by Morgan [[117](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B117)] at 1.4 to 2.8 mm in length when fully extended (Figure [(Figure4).4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F4/)). Vance Tartar carried out a systematic program of intra-species grafting to discover the limits of its recovery of normal form and of its capacity for cell division. Comprehension of Tartar's findings requires awareness of three basic facts about *Stentor*: first, 'that cell shape is an expression of the cortical stripe pattern' ([[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)], p. 211), which in turn is closely associated with the pattern of ciliary rows that are interdigitated among the stripes; second, that *Stentor* is capable of maintaining one, two or three (but not four or more) parallel sets of major cortical landmarks (oral structures and oral primordia) without severely compromising its normal cell form; and third, that the consistency of its internal cytoplasm makes it possible for an experimenter, with some skill and practice, to fuse whole stentors together in any orientation, or to graft parts of one stentor onto another stentor [[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)].



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-4.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-4.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-4.jpg)

[Figure 4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F4/)

**A descriptive diagram of Stentor coeruleus**. All of the features shown are on the cell surface, except for the macronuclear nodes, the micronuclei, and the contractile vacuole, all three of which are located just beneath the surface. The cortical fibrillar system, not shown in the diagram, is located within the clear stripes between the granular (pigmented) stripes. From Figure 1 of [[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)]. Image courtesy of Biodiversity Heritage Library. [http://www.biodiversitylibrary.org](http://www.biodiversitylibrary.org/)

The overall result of Tartar's analysis of 272 combinations of whole stentors grafted together in random orientations is best summarized in Tartar's own words: 'Fusion masses of two to four stentors were generally capable of recovering fully or approximately the monaxial, normal body shape and of dividing thereafter...to give single, doublet, and triplet progeny. In the larger grafts (involving random fusion of more than four whole stentors) both shape recovery and cell-division were lacking. ...the masses were unable to even begin fission; only artificial cutting up of the masses into approximately normal volumes produced normal singles' ([[119](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B119)], p. 564). These fusion masses (which could not feed) 'lived for about the same period of time as starved controls' ([[119](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B119)], p. 569) implying that their failure was most likely due to starvation resulting from their inability to feed rather than to anoxia resulting from their increased mass and reduced surface-to-volume ratio. This inability to feed was a consequence of the inability of these large masses of artificially fused stentors to form normal oral structures, for reasons to be explored below.

Further analysis revealed that surgical manipulation of the cortical pattern itself could influence the recovery of normal form. Microsurgical disarrangement of large blocks of cortex resulting from grafts of large portions of stentors in unnatural arrangements often brought about bizarre structural outcomes, whereas stentors demonstrated 'an astonishing capability...to regenerate and to reconstitute the normal, orderly arrangement of the ectoplasmic pattern...after all of the complex ciliary, contractile, conductive and other differentiations of the ectoplasm have been cut into tiny pieces scattered at random' ([[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)], p. 224). Further, such thorough disorganization ('minceration') of the cortex of *Stentor* increased the upper size limit for attainment of normal form: a minced six-mass (a group of six stentors artificially fused together followed by random slicing into the cell cortex of the fusion mass using a fine glass needle) 'had succeeded, as un-minced six-masses do not, in reconstituting a doublet with a single-cell shape' ([[120](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B120)], p. 200) (Figure [(Figure5).5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F5/)). Even a 25-mass (that is, 25 stentors artificially grafted together), after minceration, could manage a partial recovery of normal form, although with no oral differentiation (see below).



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-5.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-5.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-5.jpg)

[Figure 5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F5/)

**Random slicing into the cell cortex ('minceration') facilitates the integration of a large fusion mass**. **(a)** A graft of six whole stentors fused together, minced, plus an implanted oral apparatus. **(b)** After two days, four small oral primordia were formed at irregular sites. **(c)** By the sixth day, the fusion mass reconstituted a doublet with normal oral structures and a cell shape resembling that of a normal single cell. From Figure 12 of [[120](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B120)].

These observations appear paradoxical until one realizes that the graft-fusions were, as Tartar pointed out, mostly random. Thus, an un-minced fusion complex had large blocks of normally juxtaposed cortex rearranged in a coarse crazy-quilt disorder. The minceration effectively made the pieces of the quilt much smaller. These small pieces then could rotate on the fluid endoplasm, and later come into alignment with other such pieces, with selection probably favoring homopolar alignments, thereby enabling a gradual reconstruction of a coherent and fairly normal cortical pattern [[120](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B120)].

But then what is the basis of the upper size limit? Here we need to introduce one further element of *Stentor* lore. That is the notion of gradients. The German investigator Gotram Uhlig, who carried out his own experimental analysis of *Stentor* morphogenesis independently of Vance Tartar in the 1950s, explained most of his results on the basis of two interacting gradients, one basal-apical and the other circumferential [[121](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B121)]. Tartar subsequently adopted Uhlig's basal-apical gradient to account for certain otherwise inexplicable results of one of his own experiments [[122](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B122)]. The relevance of this postulated gradient in the current context is that one of its principal expressions is the induction of mouthparts at the posterior end of the oral apparatus of *Stentor* as it develops within the primordium site shown in Figure [Figure44](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F4/).

Returning to the *Stentor*-masses resulting from the fusion of whole stentors in random orientations, minced six-masses were able to form normal oral apparatuses with mouthparts (oral pouch and gullet, Figure [Figure4),4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F4/)), whereas minced 25-masses were not; they instead produced 'two garlands of adoral cilia without cytostomes' (that is, membranellar bands lacking the oral pouch and gullet) superimposed on a rough approximation of the normal *Stentor* form ([[119](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B119)], p. 559). Failure to form proper mouthparts within such 'garlands' was typical for the large fusion-masses, which Tartar attributed to 'the presence of numerous cell axes running in random directions and canceling each other in their polar influences' ([[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)], p. 215). But failure would also be expected if the sheer size of the large *Stentor-*masses rendered them unable to reconstitute a normal basal-apical morphogenetic gradient. Recalling that if the large masses were cut into smaller pieces they could then regenerate normal single stentors, one may ask whether the insuperable dilemma faced by these large masses is due to their excessive structural complexity or to their large size or to some combination of the two. This issue may be ripe for re-investigation with modern means for visualizing cytoskeletal organization.

Vance Tartar did not supply scale-bars for his published drawings of operated stentors. While he acknowledged his failure to obtain 'super-giant normal stentors' ([[119](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B119)], p. 567), the normal-appearing doublet that emerged from a fused and minced six-mass (Figure [(Figure5)5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F5/)) must nonetheless have been at least transiently larger than a normal *Stentor*. Assuming an extended length of a normal *Stentor* in the order of 2 mm, I would then estimate a maximum *linear* (basal-to-apical) dimension for form regulation in *Stentor coeruleus* at somewhere close to 3 mm.

#### A comparative analysis: early cleavages in amphibians that lay extra-large eggs

In animals, the largest dividing cells are eggs. The largest totally cleaving (holoblastic) eggs are found among amphibians. The most familiar frogs (such as *Rana pipiens* and *Xenopus laevis*) and salamanders (such as *Ambystoma mexicanum* and *Triturus*) lay eggs with a diameter between 1 and just over 2 mm [[123](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B123),[124](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B124)]. However, other species found in both of the two major amphibian orders (anurans and urodeles) produce eggs that range up to 10 mm in diameter. This large size of the egg is associated with direct development and/or parental care [[125](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B125),[126](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B126)]. Unlike the situation with Tartar's grafted stentors, we know that even the largest eggs somehow manage to complete their development. Nonetheless, we can still ask whether their early cleavages remain regular and orderly as egg diameter increases. The simple answer is that they do not.

The pattern of cleavage has recently been investigated in two large-egged amphibian species: one marsupial frog and one land-dwelling salamander. Early cleavages in the marsupial frog *Gastrotheca riobambae*, which has an egg diameter of 3 mm, are holoblastic, extremely slow, commonly asynchronous, and frequently asymmetric in that they do not cut the egg into equal halves. Hence, when different eggs at the same cleavage stage are viewed from above the animal pole, each one has a different pattern of blastomeres, and the number of complete blastomeres at any given time is commonly not in the series of 2n[[127](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B127)]. The plethodontid salamander *Ensatina eschscholtzii*, the largest appropriately studied amphibian egg, has a 6 mm diameter, and has taken a further step in the direction of meroblastic (partial) cleavage. Early cleavages (beyond the first two) are extremely irregular, and 'cleavage initially occurs only in the animal pole, with no cleavage furrow visible in the vegetal pole until about the 16-cell stage' ([[124](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B124)], p. 3). This implies that progression of cleavage furrows from the animal into the vegetal region is either very slow or delayed, probably due to the high concentration of yolk in the vegetal hemisphere.

Collazo and Keller [[124](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B124)] discuss the modifications of cleavage in large-egg amphibians in the light of a distinction proposed by SJ Gould, between 'historical' and 'formal' developmental constraints. Historical constraints are ones that depend upon contingencies of ancestry and descent. Formal constraints exist independently of ancestry and are dictated by physical principles or restrictive structural relationships [[128](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B128)]. Collazo and Keller attribute most of the modifications of early development found in large-egged salamanders to historical constraints, because these are diverse in different lineages. However, they make an exception for the effect of egg size on early cleavage patterns, which they attribute to a formal constraint in the Gouldian sense. In their words, 'The fact that the asymmetries and asynchronies in early cleavage seen in these four species (the three others are from early 20th Century descriptions of large-egged salamanders) are qualitatively similar and that these species represent two disparate salamander families suggests that large egg size and not phylogenetic relationship accounts for the differences in development from amphibians with smaller eggs' ([[124](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B124)], p. 8).

To a first approximation, the threshold in linear dimension between regularity and irregularity of amphibian cleavage appears to be somewhere between 2 and 3 millimeters - similar to the size threshold in capacity of *Stentor*-masses to regulate to the normal *Stentor* form (and concomitantly the normal capacity to divide). This might be a general size-limit in the capacity for a well-organized binary cell division.

#### Divergent organization: outside-in versus inside-out

Even if we accept that a similar upper size threshold exists for normal cell division in *Stentor* and in amphibian (and fish) eggs, it could still be that the similarity in these size thresholds is coincidental. This is especially likely because the respective roles of the cortex and the endoplasm in the cell's accommodation to large cell size appear to be opposite in the two types of cells.

In *Stentor* as in other ciliates, the cortical layer is structurally the most highly organized part of the cell. That is where the granular pigment stripes and the intervening cortical fibrillar system, including basal bodies, cilia, and accessory microtubular bands, are located. The nodes of the macronucleus adhere to the cortical layer and several small micronuclei are nearby [[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)] (Figure [(Figure4).4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F4/)). An ultrastructural study of a closely related ciliate *(Blepharisma)* has shown that the mitotic division of the micronucleus is closed, with no trace of centrioles, centrosomes or astral fibers [[129](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B129)]. Tartar was able to remove 'practically all the endoplasm [of *Stentor*] by vigorous pipetting', after which the eviscerated stentors could 'regenerate and fill out the cell shape within a day' ([[118](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B118)], p. 108). Cell division in *Stentor* is a process that, to a large extent, is driven by the longitudinal growth and transverse segmentation of the cortical pattern [[130](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B130)].

The division of a fertilized frog egg could hardly be more different. Frog eggs are roughly spherical and have no obvious cortical differentiations, apart from the pigmented cap on the animal hemisphere [[131](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B131)]. The division apparatus is entirely internal. The mitotic spindle is small relative to the large size of the egg and becomes located deep within the cell (Figure [(Figure6,6](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F6/), left). The asters, on the other hand, are dynamic structures that re-form at telophase of each of the early divisions, expand tremendously, and while expanding become centered by dynein-mediated pulling forces that act on the astral microtubules even before these microtubules touch the cell surface. Cell division furrows then ingress 'where the interaction zones between telophase asters touch the cortex' ([[89](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B89)], p. 2043). Thus, while the cortex is involved passively in the determination of the location of the fission zone (and actively in its subsequent ingression), the earlier dynamic processes, including the sensing of cell volume, are all endoplasmic. These mechanisms, based on the balancing of pulling forces from multiple locations in the cytoplasm rather than from the cortex, were proposed as adaptations that would allow asters to function properly in large-sized amphibian eggs [[97](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B97)], yet they also appear to function in the smaller eggs of the sea urchin and of the nematode worm *Caenorhabditis elegans* [[75](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B75),[81](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B81),[86](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B86)] (also see the contribution from Dr Kimura in this Forum article).



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-6.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-6.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-6.jpg)

[Figure 6](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F6/)

**Accommodations must be made**. The one cell *Xenopus* embryo (left) is over 1 mm across, while cells within the embryo several hours later (right) are closer to 50 microns. Mitotic spindles are shown in red and chromosomes in yellow. Complex mechanisms have evolved to allow for cellular functions such as cytokinesis and mitosis to be effective in cells of diverse sizes. Images are courtesy of Martin Wühr (Harvard).

These studies were carried out on the egg of *Xenopus laevis*, which at a diameter of 1.2 millimeters is large as a cell and even as an egg, yet smaller than the eggs of many other frog species [[126](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B126)]. Therefore, one wonders whether the irregularities of cleavage that emerge as amphibian eggs get larger are in some way related to size limitations in the effective functioning of the centrosomal-centering and cleavage-site determining mechanisms that operate so efficiently in the *Xenopus* egg. This problem is discussed by Kimura in this Forum, and is illustrated in Figure [Figure3.3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/). The postulated 'nonsymmetric cell division' resulting from the failure of centrosomes to reach the cell center (Figure [(Figure3,3](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F3/), top center) is reminiscent of the irregular cleavage patterns observed in the very large eggs of the frog *Gastrotheca riobambae* [[127](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B127)] and the gigantic eggs of the salamander *Ensatina eschscholzii* [[124](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B124)].

In view of these major differences, we can also wonder whether the size limits of the two largest cleaving cell types, the large ciliate *Stentor coeruleus* and the even larger amphibian eggs such as those of *Ensatina eschscholtzii*, have anything at all in common. I think that they just might. In cleaving eggs, there are good reasons to believe that size limits are based on properties of microtubules (see above, as well as the contribution from Dr Kimura in this Forum article). While a consideration of microtubular systems of ciliates is beyond the scope of this contribution, these systems are known to be abundant and in part dynamic over the cell cycle, yet have been little investigated in the largest ciliates such as Stentor. It is just barely possible that there might be some underlying limit to the spatial extent over which microtubule-based cytoskeletal systems can organize and then reorganize themselves in the absence of internal cell boundaries. Further structural and molecular investigations of large minced *Stentor* grafts and the largest amphibian eggs might yield some interesting and unexpected insights into these and perhaps other unanticipated questions.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## Size matters, but in animals so does shape

### John Wallingford

I am nearly two meters tall, so the neurons linking my toes to my spinal cord are quite enormous. These neurons are well over twice the size of those belonging to my four year old daughter, even though her skin fibroblasts are probably about the same size as mine. This anecdote, unscientific though it may be, serves to illustrate two key facts. First, that in addition to the many problems of cell size control faced by unicellular organisms, animals face the added challenge of establishing and maintaining cell type-specific cell sizes. And second, that the need to control cell size over developmental time presents an additional hurdle.

The control of cell size in animals is of course a wide-ranging topic, and it is no surprise that key players in cell size control in unicellular organisms are also key players in animal cell size control. Genetic studies in *Drosophila* have revealed the key role of cell cycle regulators in controlling cell size, and the phosphoinositide 3-kinase pathways are also widely studied for their link to cell size in mammals [[132](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B132),[133](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B133)]. However, given the deep conservation of such mechanisms, it is perhaps more interesting in this forum to discuss some less well-known, cell type-specific problems that arise at the interface of cell size control and development. In this respect, a consideration of developing amphibians provides some illuminating vignettes.

Some of the pioneering studies for the link between cell size and cell proliferation in animals were performed in salamanders, where Fankhauser noted that the increase in cell size in heteroploid animals was compensated for by decreases in cell numbers. Thus, he found that the salamanders and their constituent organs were all roughly the same size, be they diploid, triploid or even pentaploid [[134](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B134)]. Such compensatory effects are widespread in animals, as reflected by the more recent genetic studies in *Drosophila*, for example [[132](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B132)].

Like most animals, amphibians develop externally and without ongoing maternal nutrition, and so their eggs are packed with yolk. The enormous size of the one-cell frog embryo (>1 mm across [[135](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B135)]) is therefore a crucial facet of its lifestyle, but it also presents a problem. During cell division, these large cells must deploy specialized mechanisms for generating the enormous amounts of new plasma membrane to build the >500 micron-long nascent cleavage furrow [[136](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B136)]. Likewise, the mechanisms of mitosis have been modified to achieve proper chromosome separation in such a gigantic cell [[89](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B89),[97](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B97)], even though the size of their spindles remains surprisingly small, capped apparently by an upper physical limit [[137](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B137)] (Figure [(Figure6).6](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F6/)). During these early stages, cell division is uncoupled from cell size [[138](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B138)], but these modifications - however crucial to the early embryo - are quickly abandoned. By the 12th division, cells are a far more reasonable approximately 50 microns in diameter and links between cell size and cell division are put in place [[138](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B138)].

At this same time, another developmental landmark serves to illustrate the importance of cell size: after 12 divisions, *Xenopus* embryos engage the zygotic transcriptional machinery for the first time [[139](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B139)]. As in many other animals, this onset is determined by a nuclear-to-cytoplasmic volume ratio [[140](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B140)]. This ratio must necessarily be influenced not only by cell size, but also by nuclear size, so it is noteworthy that nuclear size, like organelle size generally, is not a simple reflection of overall cell size. Rather, recent *in vivo* studies in frog embryos combined with *in vitro* studies exploiting embryo extracts have identified factors in the cytosol that are crucial to the control of nuclear size [[141](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B141)]. These cytosolic factors are even more crucial than is ploidy [[141](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B141)], a result that in many ways parallels findings in yeast [[60](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B60)].

Similar experiments suggest that mitotic spindle length is also dependent on cytosolic factors [[142](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B142)]. In the smaller cells of later stage embryos, spindle length scales with cell size [[137](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B137)], and this scaling requires input from the actin cytoskeleton [[143](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B143)]. In larger cells, spindle length does not scale with cell size, and even in cytoplasm extracts *in vitro*, where spindles cannot be constrained by any physical cue, there are spindle-intrinsic cues that set a strict upper limit on length [[137](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B137),[144](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B144)]. Collectively, these results not only illustrate some of the recent advances in our understanding of organelle and cell size in embryos, but they also highlight a general gap in our understanding at the intersection of developmental and cell biology: we have a fairly detailed picture now of many fundamental cell biological processes, but much of this picture is drawn from studies of relatively few cell types, many of which exist only in culture. Though comparatively sparse, *in vivo* studies consistently show that these fundamental processes vary from cell type to cell type in animals, but the factors controlling such cell type-specific modifications remain for the most part poorly defined.

Finally, there is one issue of cell size control that may be unique to animals, and that is the impact of cell size on cell movement. Large-scale movements of individual cells are central to animal morphogenesis, and the last decade has seen huge leaps forward in our understanding of the molecular control of force generation during animal morphogenesis [[145](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B145),[146](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B146)], but we know very little about how cell size influences these processes. This fundamental question was articulated by Fankhauser himself, who noted that radical changes in cell shapes were needed in order to generate normally shaped organ structures out of the much larger cells in heteroploid animals. One example he gave was the developing kidney tubule, where five or six cells of roughly columnar shape spanned the circumference in normal diploid animals. Only two of the larger cells in a pentaploid animal enclosed the tubule, and these cells were flattened and curved such that tubule diameter remained similar to that in diploids (Figure [(Figure7).7](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F7/)). This finding suggests that regulatory mechanisms are in place to sense cell size and adjust cell morphology accordingly in order to maintain tissue structure [[134](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B134)]. Conversely, mechanisms also exist to allow larger cells to form a larger but morphologically normal kidney, a situation called compensatory renal hypertrophy [[147](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B147)]. Such compensation commonly occurs in one kidney when the other is somehow compromised.



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-7.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-7.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-7.jpg)

[Open in a separate window](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F7/?report=objectonly)

[Figure 7](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F7/)

**Accommodations must be made again**. Animals have evolved mechanisms that maintain tissue size in the face of changing ploidy. These images from Fankhauser illustrate this point. Kidney tubules of the same size are constructed by larger and larger cells as ploidy increases, and cells must change their shapes so that the tubule diameter can remain constant. In the haploid animal, many nuclei can be observed around the tubule circumference, and so these cells have a columnar morphology. In the pentaploid animals, only one or two nuclei can be observed, and these much larger cells are flattened and squamous in order to enclose the same tubule diameter.

Recent molecular studies also provide clues to the interaction between cell size control and morphogenesis. For example, live imaging suggests that kidney tubule diameter is controlled in part by cell rearrangements, and when these are disrupted, tubules become dilated and cystic [[148](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B148),[149](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B149)]. Similar cystic phenotypes have been linked to the Hippo pathway, a key signaling mechanism governing cell division and organ size control [[150](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B150),[151](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B151)]. Likewise, the same genetic pathways that govern cell size control in normal development also govern renal hypertrophy [[152](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B152),[153](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B153)]. So here again, detailed cell biological studies performed *in vivo* will be central to our efforts to understand the tangled interactions between cell size and morphogenesis in animals.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## Thinking inside the wooden box - classic views of cell size control in plants

### Virginia Walbot

Historically, botanists quantified various cellular shape and volume parameters and discovered a very tight correlation between nuclear DNA content (the C value), nuclear volume, and cell volume in meristematic cells with small vacuoles [[154](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B154)]. Over a diverse range of C values, including exemplar monocots with giant genomes, mid-range genome size monocots and dicots, and *Arabidopsis thaliana* with a tiny genome, the log cell volume is linearly related to log nuclear volume with a correlation of 0.99. There are numerous 'ploidy' series within species (or very close relatives) in nature and among horticultural derivatives. *Ficus* spp. trees, ubiquitous decorations in hotel and airport lobbies, are diploid, tetraploid, or octoploid. These ploidy levels are readily distinguished by comparing the size of the epidermal guard cell pairs or other epidermal cells. From a few centimeters distant, however, the trees are indistinguishable in architecture and leaf size (V Walbot, personal observation) despite the obvious fact that octoploid leaves contain fewer, larger cells.

In the Introduction to this Forum, Wallace F Marshall opines that cell physiology depends on cell size. This is true, perhaps, in cuboidal animal cells in which relative surface area declines with volume and the cytoplasm is served by a 'smaller' surface area. Because plant cells have vacuoles, however, gigantic diploid cells with very large vacuoles will have a tremendous surface area, and such cells will actually experience a much higher surface area per unit cytoplasm than a smaller cuboidal cell. In a ploidy series or in an organ with cells of various ploidy levels, the higher ploidy cells can have an even more favorable surface to cytoplasmic volume relationship.

Observations on living plants indicate that growth and most morphology do not depend on the number of cells or their size within an organ. A very striking observation concerns lethally irradiated seeds: upon germination the pre-existing cells of the embryo enlarge, generating a seedling that is remarkably normal despite the lack of cell division [[155](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B155)-[157](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B157)]. Not so surprising for the first leaf, which contained almost the full complement of cells, but each successive leaf starts with fewer cells yet achieves near normal morphology despite ridiculously large cells and abnormal anatomy. In a separate study, the same researchers applied colchicine to inhibit cell division in roots, yet single-celled lateral root primordia initiated in the normal location and grew in a manner paralleling the normal developmental pattern of multicellular lateral roots [[158](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B158)].

Yet, plant development is highly regular, and leaves and roots of similar size with approximately equivalent numbers of cells are produced by a population of seedlings. This regularity of developmental pattern is particularly high in reproductive organs such as the anther, in which stereotyped cell division patterns establish the tissues, and there is a high degree of similarity in cell numbers and volumes in different anthers [[159](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B159)]. For example, anther developmental outcome in maize is extremely similar in different inbred lines, although developmental mechanism may differ: in the inbred W23 the secondary parietal layer makes a physically symmetric periclinal division to establish the middle layer and tapetal cell layer while in inbred A619 this division is highly asymmetric. At the conclusion of cell patterning and growth, however, the differentiated middle layer and tapetal cells are virtually identical in these two inbred lines. This regularity of development is the basis for genetic screens to identify mutants with perturbations in cell enlargement or cell division. Despite the large number of mutants available and many measurements of growth parameters, the relationship between cell division, cell expansion, and their balance to establish cell size remains mysterious. Moving up to the scale of a tissue or organ, the cellular composition (number of cells and their sizes) can vary widely, depending on ploidy or treatments that modulate cell division and expansion.

Emergent patterns during plant growth is a topic that fascinated Alan Turing. This brilliant mathematician pondered the Fibonacci series intrinsic to spiral botanical patterns (pinecones, petals, and so on), although unfortunately nearly all of his botanical insights remained as unpublished manuscripts and notebooks. Thus, it is gratifying that today in addition to cytological and genetic study of plant development, mathematical modeling is enjoying a renaissance in plant biology, and the fundamental problem of growth control is one attractive target. Improvements in cell imaging - principally the use of confocal imaging and the use of *in vivo* fluorescent cell type markers - has generated copious data on cell numbers, sizes, and changes in dimensions over developmental time. These new datasets have sparked a renaissance in the application of mathematical modeling to plant growth. The integration of observation, computation, and simulation represents a new frontier in growth analysis, with cells at the heart of all three ways of thinking about tissue growth. In the next section, our current state of knowledge, tool kit, and challenges for these three paths are integrated and critiqued.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## Plant cell size control: all things considered

### Xian Qu, Adrienne HK Roeder

Plant biologists have long wondered about the mysterious phenomenon called 'compensation': when the cell number is decreased as the result of a mutation, the cell size increases, leading to the production of organs with nearly normal area [[160](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B160)]. This phenomenon raises two basic questions for plant biologists: how is the size of a plant organ controlled and how is the size of plant cells controlled? Although these interrelated puzzles have been extensively studied for many years, neither is fully understood. Two processes contribute to size control during organogenesis: cell division and cell growth. Cell size growth in plants is driven by increase in mass (reflecting macromolecular synthesis) and increase in volume (primarily through expansion of the cell wall and the central vacuole). The relationship between cell division, cell size, and organ size remains controversial, as illustrated by the contradictory hypotheses put forward in two recent computational models [[161](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B161),[162](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B162)]. Here we will examine the contributions of cell cycle regulation, ploidy, macromolecular synthesis of cytoplasm, cell wall expansion, and developmental regulation in the control of plant cell size, focusing on examples from *Arabidopsis*.

Changing cell cycle regulator activity can alter the length of time that cells spend in growth phases G1 and G2 before dividing, thus affecting cell size. In *Arabidopsis* leaves and cultured cell lines, overexpression of *CYCLIN D3;1* (*CYCD3;1*) triggers a quick transition into the mitotic cycle, reducing the proportion of cells in the G1 phase of the cell cycle, and decreasing cell size [[163](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B163),[164](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B164)]. In contrast, slowing division with a dominant negative form of the *CYCLIN DEPENDENT KINASE A;1* (*CDKA;1*) increases final cell size [[165](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B165)]. Likewise, expressing the CDKA;1 inhibitor *KIP RELATED PROTEIN1* (*KRP1*) under the control of an epidermis-specific promoter results in slower epidermal division and somewhat increased epidermal cell size [[162](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B162),[166](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B166)]. In contrast, overexpressing either *APC10* or *CDC27*, which are two subunits of the anaphase-promoting complex/cyclosome (*APC/C*), increases cell division rates without decreasing the final cell size [[167](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B167),[168](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B168)]. Thus, cell size can be altered by some cell cycle regulators, but is unaffected by others.

In many organisms, including plants, a strong correlation exists between ploidy, nuclear size, and the volume of the cytoplasm [[169](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B169)-[171](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B171)]. In young plant cells with small vacuoles, the cytoplasm fills the cell, so that ploidy correlates with cell volume. In contrast, in mature plant cells, the large central vacuole fills the volume, and the cytoplasm is restricted to a thin layer on the surface of the cell, such that the ploidy correlates with the surface area [[172](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B172)].

Polyploid cells are found outside of polyploid plants; diploid plants commonly contain polyploid cells produced through endoreduplication - a variant of the cell cycle in which cells stop dividing and instead continue to grow and replicate their DNA [[171](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B171)]. Endoreduplication causes an increase in both cell size and ploidy [[172](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B172)] and is often associated with specialized cell types. Endocycles and mitotic cell cycles can occur simultaneously during development in neighboring cells [[162](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B162)] and endoreduplication is a mechanism for cell enlargement. Many alterations inhibiting the cell cycle regulatory machinery cause increased endoreduplication in organs. Although the exact trigger that causes an individual cell to endoreduplicate is unknown, one speculation is that the cell needs to exceed a prior cell size checkpoint. One finding supporting this hypothesis is that overexpression of either of two D-type cyclins, *CYCD2;1* or *CYCD3;1*, induces smaller cell sizes and blocks endoreduplication [[163](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B163),[173](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B173)]. Once the putative cell size checkpoint is passed, endoreduplication can be switched on by various mechanisms. One such mechanism is reduction of CDKA;1/CYCLIN B activity to a level that fails to initiate mitosis but is still able to drive replication of DNA. For example, mitotic CDKA;1 activity can be reduced by elevated activity of cyclin-dependent kinase inhibitors of the KRP and SIAMESE families [[174](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B174),[175](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B175)]. The SIAMESE protein has CYCLIN and CDKA;1 binding sites, and the protein can inhibit mitosis and stimulate endoreduplication in the *Arabidopsis* hair cells (trichomes) [[176](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B176),[177](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B177)]. A second trigger of endoreduplication is ectopic DNA replication. *CDC6* catalyzes assembly of the ORIGIN OF REPLICATION (ORC) complex, enabling DNA replication and overexpression of *CDC6* increases endoreduplication in *Arabidopsis* [[178](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B178)].

In plants, the ratio of the ploidy to the cytoplasm remains constant, suggesting that cytoplasm can influence cell size as well. Cytoplasm increases through macromolecular synthesis. For example, *Arabidopsis* EBP1 is related to human ErbB-3, an epidermal growth factor receptor binding protein that enhances translation. Overexpression of the *EBP1* gene in *Arabidopsis* increases cell growth [[179](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B179)], implying that increased protein synthesis correlates with increased cell size. Similarly, inhibiting protein degradation by mutation of the 26S proteasome subunit *REGULATORY PARTICLE AAA-ATPASE 2a* (*RPT2a*) [[180](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B180)] produces larger cells in *Arabidopsis* leaves [[181](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B181),[182](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B182)], showing that optimizing proteasome activity levels is important for cell size control. As in many other organisms, macromolecular synthesis is regulated in *Arabidopsis* by the TARGET OF RAPAMYCIN (TOR) pathway [[183](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B183)]. The pathway regulates cell growth and metabolism in response to growth factors, nutrients, energy, and environmental conditions in yeast and mammals [[184](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B184),[185](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B185)]. The *Arabidopsis* genome encodes only one *TOR* gene, and it regulates cell size, translation initiation, and ribosomal RNA synthesis [[183](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B183),[186](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B186)].

Cell wall expansion is a major mechanism controlling plant cell size. The plant cell wall greatly confines the enlargement of plant cells because it consists of a rigid mesh of complex polysaccharides and a few structural proteins [[187](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B187)]. The cell wall grows by repeated stretching and polymer reconnection (termed stress relaxation): turgor pressure constantly exerts a stretching force on the cell wall, and the force is alleviated when polysaccharide polymers rearrange their interconnections to take on the new shape [[188](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B188)]. To maintain cell wall integrity, new polymers are synthesized simultaneously and added to the growing wall. Thus, proteins that modify the interconnections between polysaccharide polymers regulate plant cell growth. The *EXPANSIN* (*EXP*) family is one class of plant proteins that are thought to loosen cell walls by weakening the binding of polysaccharide polymers to one another [[189](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B189)]. In fact, adding active EXPANSIN proteins to dead cell walls is sufficient to cause their rapid extension [[190](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B190)]. Overexpression of *EXP10* in *Arabidopsis* under the control of its own promoter results in larger leaves containing larger cells [[191](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B191)]. The extensibility of the cell wall is not uniform in all directions, and is limited by the reinforcement of the cellulose microfibrils embedded in the wall. The primary direction of expansion, and ultimately the shape of the cell, depend on the orientation and alignment of the cellulose microfibrils, which are controlled by the cytoskeleton just inside the plasma membrane of the cell. In vascular plants, a cellulose microfibril is synthesized by a cellulose synthase complex of integral plasma membrane proteins [[187](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B187)]. The cellulose synthase complex tracks through the plasma membrane along the cortical microtubules in such a way that microtubules determine the orientation of the new cellulose microfibrils in the cell wall, and thus control the direction of structural reinforcement [[192](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B192)]. In *Arabidopsis*, both microtubule- and microfilament-associated proteins facilitate normal cell expansion in different organs [[193](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B193)-[195](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B195)]. The cell wall presents a second challenge to plant cell growth in that the walls of neighboring cells are tightly connected. Because plant cells generally do not slip relative to one another, the growth of neighbors is coordinated along their adjoining walls. Nonetheless, heterogeneity of cell growth is not only still possible but common in individual cells, because portions of the cell wall in contact with different neighbors can grow with different rates [[196](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B196)].

The relationship between organ growth through turgor-induced cell wall expansion and organ growth through cytoplasm biosynthesis and cell division is unclear. As mentioned above, plant cells have a tremendous ability to increase their cell wall expansion to compensate for decreased cell division, resulting in an organ with normal size but extremely large cells. However, compensation fails when too few cells are available. Perhaps this was best illustrated by experiments in which cell division was completely blocked in wheat seedlings by gamma irradiation of the grains. Remarkably, the leaf primordia of these gamma plantlets still grow and produce first foliage leaves with the correct shape [[155](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B155),[197](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B197)]. However, after ten days, the gamma irradiated plant leaves are only about 15% as long as unirradiated plants, despite greatly increased cell expansion [[197](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B197)]. These experiments indicate that there is a limit to cell wall extensibility in living plants and some cell division is required for organs to reach their normal size.

All of these cell size control pathways can be developmentally regulated with temporal, tissue and spatial specificity to produce a variety of different cell sizes, often corresponding to specialized cellular functions. Cell size is regulated by transcription factors and co-activators such as GROWTH-REGULATING FACTOR (GRF) [[198](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B198),[199](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B199)] and ANGUSTIFOLIA3 (AN3) [[200](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B200),[201](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B201)]. Furthermore, cell size is actively patterned in different plant organs. For example, the *Arabidopsis* sepal epidermis has a broad diversity of cell sizes, from small cells with one-hundredth the length of the sepal to giant cells with one-fifth the length of the sepal (Figure [8a,b](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F8/)). This pattern forms as a result of the stochastic entry of cells into endoreduplication at different times [[162](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B162)]. The pattern is also regulated by the epidermal specification pathway, which promotes the formation of giant cells [[116](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B116)]. Giant cells are found on the back (abaxial) epidermis of *Arabidopsis* sepals and leaves, whereas the epidermal cells in the petal blade do not endoreduplicate and consequently they have a uniform small size.



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-8.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-8.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-8.jpg)

[Open in a separate window](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F8/?report=objectonly)

[Figure 8](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F8/)

**The diversity of cell sizes in the Arabidopsis sepal epidermis**. **(a)** A scanning electron micrograph (SEM) of the sepal epidermis shows large giant cells are interspersed between smaller cells in a range of sizes. **(b)** A confocal maximum intensity projection of the sepal epidermis in which the cells are outlined in red by the plasma membrane dye FM4-64 revealing the variation in cell sizes. The nuclear size shown in green (*ML1::H2B-mYFP*) corresponds with the ploidy. The correlation between cell size and ploidy is evident: large cells have large nuclei indicating that they have undergone endoreduplciation whereas small cells have small nuclei indicating that they have remained diploid. Scale bars represent 50 μm.

In summary, cell size control in plants is a highly dynamic and complicated process involving multiple biological pathways (Figure [(Figure9).9](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F9/)). Although many individual cell size regulators in each of these pathways have been discovered, a future challenge will be determining how these pathways integrate to form a complete cell size control network. Two recent advances are likely to guide the construction of such a network [[202](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B202)]. First, live imaging of cells in developing organs will allow us to determine the exact timing and detailed mechanism through which each player affects cell division, cell growth, and cell expansion. For example, the transcription factor JAGGED (JAG) regulates proliferation to control lateral organ shape and size in *Arabidopsis* sepals and petals [[203](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B203),[204](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B204)]. Recently, by using three-dimensional live imaging, Schiessl *et al*. [[205](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B205)] discovered that *JAG* regulates the transition between tight coordination of cell volume with initiation of S phase of the cell cycle in the stem cells of the meristem and loose coordination in the initiating organ primordium. Ectopic expression of *JAG* in the floral meristem stem cells bypasses this tight cell size checkpoint, resulting in smaller cells entering S phase [[205](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B205)]. Second, computational modeling will allow us to predict the cumulative effect of multiple pathways acting simultaneously and feeding back on one another [[206](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B206)]. For example, while a diagram can be used to conceptualize the increase in cell size caused by one cell entering endoreduplication earlier than its neighbors [[207](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B207)], a computational model can expand this analysis to about 1,400 cells entering endoreduplication or dividing at stochastic times to pattern the entire sepal [[162](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#B162)]. Such an integrated understanding may finally show how plant cells increase in size to compensate for decreased cell numbers to achieve consistent organ size.



[![An external file that holds a picture, illustration, etc. Object name is 1741-7007-10-101-9.jpg](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/bin/1741-7007-10-101-9.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click on image to zoom&p=PMC3&id=3522064_1741-7007-10-101-9.jpg)

[Figure 9](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/figure/F9/)

**Overview of cell size control in plants**. Plant cell size is determined by the growth of the cell versus its division. Cell growth is determined by macromolecular synthesis as well as expansion of the cell wall. Future challenges include untangling the regulatory network between the various pathways regulating the cell cycle, endoreduplication, the cell wall, and synthesis of cytoplasm, to elucidate how this crosstalk determines the ultimate cell size.

[Go to:](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522064/#)

## Acknowledgements